src/components/LinksSection.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from "prop-types";
+
+const LinksSection = ({ className, linksList }) => {
+  return (
+    <div className={className}>
+      <h3 className="pb-10 text-2xl font-bold uppercase font-heading">Links</h3>
```

Comment:
semantically I think `h2` might make more sense here, this is how I'm imagining the page is structured, for example

```
h1: Arise Indonesia (title)
  h2: The Recap
  h2: The Moments
  h2: Our Speakers
  h2: The Summary
    h3: Program
    h3: What Did You Think About Arise Indonesia?
  h2: Links
  h2: Arise Asia
  h2: Arise Indonesia (footer)
```

I also added some utility classes that resize the text by viewport width in `index.css` we can use for consistency in the design system. In the future, ideally we would have something similar for buttons and other stuff too but for now let's use either `text-h2` or `text-h3` depending on which size makes more sense